examples/scene_collection.py

```diff
@@ -1,81 +1,81 @@
 from gamebook.scene import *
 from gamebook.option import *
-from gamebook.decision import *
+from gamebook.nextscene import *
 from gamebook.helpers import openscrn
-from gamebook.chance_option import ChanceOptions
+from gamebook.options import Options
 
 
 class FleeScene(Scene):

```

Comment:
I don't understand why all of these are classes and not instances of a class?

Reply:
it is a very old file, way back when we started and the idea behind it is no longer relevant, i keep it for having references of scenes.
i can change them to instances, but what do you say we turn it to a plain text file? because i dont really need it as a python file. we could just have it for referance

Reply:
If you want to keep it for reference, add a directory called: docs, later when you will write documentations it will replace the files there.

Reply:
But remove it from the project source (it also reduces your coverage if it stays for no reason).